.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish package to PyPi
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  push:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.5
```

Comment:
Can we change this to 3.8?

Reply:
it is being used by `edx-proctoring` that is published to PyPi using python 3.5

Reply:
I'm not aware of any problems with using a package created and uploaded with a newer python version in an older one.  I'd like to minimize our usage of an unsupported python version.